packages/client/src/api/wargames_api/index.ts

```diff
@@ -386,6 +386,29 @@ export const deletePlatformType = (dbName: string, platformType: PlatformType):
   })
 }
 
+export const duplicatePlatformType = (dbName: string, currentPlatformType: PlatformType): Promise<Wargame> => {
+  return getLatestWargameRevision(dbName).then((res) => {
+    const newDoc: Wargame = deepCopy(res)
+    const updatedData = newDoc.data
+    const platformTypes = updatedData.platformTypes && updatedData.platformTypes.platformTypes || []
```

Comment:
@lilitkarapetyan - on the next two lines we find the index of the platform, then duplicate the array item at that index.

Is there a reason why we're not using the single-line `platformTypes.find`?

Reply:
I did this to not change the duplication logic and it works like duplicating channels.